third_party/shellcheck/run.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash -eu
+#
+# Copyright 2021 The Outline Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+readonly VERSION='v0.7.1'
+readonly DOWNLOAD='download'
+
+# The relative location of this script.
+dir="$(dirname "$0")/${DOWNLOAD}"
+readonly dir
```

Comment:
```suggestion
# The relative location of this script.
readonly DOWNLOAD_DIR="$(dirname "$0")/${DOWNLOAD}"
```

The `DOWNLOAD` variable is not really useful and hurts readability.

Reply:
Done